cmd/plotFunctions/plotFunctions.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/numbers"
+	"github.com/vertgenlab/gonomics/popgen"
+	"github.com/vertgenlab/gonomics/common"
+	"log"
+	"fmt"
+	"flag"
+	"math"
+)
+
+func plotFunctions(function string, left float64, right float64, bins int, outFile string, alpha float64) {
+	if function == "AfsStationarity" {
+		numbers.Plot(popgen.AFSStationarityClosure(alpha), left, right, bins, outFile)
+	} else {//here you can add more else ifs to add additional functions for plotting
+		fmt.Printf("Unrecognized function: %s.\n", function)
+	}
+}
+
+func usage() {
+	fmt.Print(
+		"plotFunctions-returns a tab separated list of function evaluations for plotting functions.\n" +
```

Comment:
did you mean to include a couple spaces around the '-'?
i was thinking something like fmt.Print("plotFunctions - returns ...") looks a little nicer aesthetically, but if that is on purpose, then scratch this comment.